<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalFolderWidget</class>
 <widget class="QWidget" name="GlobalFolderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="enabledCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pathTextBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pathBrowseBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
